04_pig_latin/pig_latin.rb

```diff
@@ -1 +1,18 @@
-#write your code here
+# It's not mine solution, it's Shanna's. But I think it's perfect and don't want to change!
```

Comment:
It is a very clean and short code block, did you guys discuss it come out with the solution together?